src/store/messages/api.ts

```diff
@@ -72,13 +73,22 @@ export async function uploadAttachment(file: File): Promise<AttachmentUploadResu
   });
   const uploadInfo: AttachmentResponse = response.body;
 
-  // note: for adding progress bar to the upload
-  // https://github.com/m3m3n70/zero-web/blob/development/src/app/api/file/index.ts#L105
+  console.log('uploadInfo', uploadInfo);
```

Comment:
Debugging?

Reply:
hell 🤦‍♂️ i had removed from local, forgot to push